websites/S/Songsterr/presence.ts

```diff
@@ -11,25 +11,28 @@ presence.on("UpdateData", async () => {
 		},
 		{ pathname } = document.location,
 		{ role } = document.querySelector("#apptab"),
-		searchvar = (<HTMLInputElement>(
-			document.querySelector("#sticky-list-header > div.Ccm27v > div > input")
-		))?.value,
 		obj: { [key: string]: string } = {
 			plus: "Viewing Plans",
-			mytabs: "Checking submitted tabs",
+			mytabs: "Checking Submitted Tabs",
 			submit: "Submitting Tabs",
 			help: "Reading FAQ",
-			howtoreadtab: "Learning how to read a tab",
+			howtoreadtab: "Learning How To Read A Tab",
 			account: "Viewing Account Settings",
 			favorites: "Viewing Favorite Tabs",
 			payment: "Buying Songsterr Plus",
 		};
 
-	if (role === "complementary")
-		presenceData.details = obj[pathname.split("/").at(-1)];
-	else if (typeof searchvar !== "undefined")
-		presenceData.details = `Searching tabs for ${searchvar}`;
-	else {
+	if (role === "complementary") {
+		if (pathname.split("/").at(-1) in obj)
+			presenceData.details = obj[pathname.split("/").at(-1)];
+		else {
+			presenceData.details = `Searching tabs for ${
+				(<HTMLInputElement>(
+					document.querySelector("div.Bt4t5 > div.Ccm27v > div > input")
+				))?.value
```

Comment:
Preferably use generic types for `querySelector` rather than casts. If possible, use non-computer generated selectors.